### PR TITLE
Specify tcp/udp protocol

### PR DESCRIPTION
**Pull request by BokilaLin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,11 @@ The super easy way to setup a [Tor](https://www.torproject.org) [SOCKS5](https:/
 
    Just note that all the connections will be terminated and need to be reestablished.
 
+## DNS over Tor
+
+- Point your router's or device's DNS server to `127.0.0.1`.
+- Go to [dnsleaktest](https://www.dnsleaktest.com) to run a test.
+
```
Comment (PeterDaveHello) [inline README.md:88]: I think the "usage" part should also be updated, and maybe separate README into another PR, as we can get this one merged very soon, and they are separated topic, thanks.

Comment (BokilaLin) [reply README.md:88]: well
I do not understand your point
so I will stop make any further contribution
just close this PR as you like

Comment (PeterDaveHello) [reply README.md:88]: I mean the [usage part](https://github.com/PeterDaveHello/tor-socks-proxy#usage) should also contain the port 53 publish, it mentioned the original 9150 only, which means the DNS port you added won't work in the example, sorry for anything unclear to you.

**Comment by PeterDaveHello** [comment]
@BokilaLin thank you for your contribution!
